<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-90</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 127); font-size:22px</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>121</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:24px; 
color: rgb(225, 225, 225);</string>
   </property>
   <property name="text">
    <string>Blood Bank</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>261</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:34px; 
color: rgb(225, 225, 225);</string>
   </property>
   <property name="text">
    <string>Forget Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 127); font-size:22px</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(225, 225, 225); font-size:16px</string>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="resetbutton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(167, 168, 167); 
color: rgb(255, 255, 255);
font-size:18px</string>
   </property>
   <property name="text">
    <string>Send Reset Mail</string>
   </property>
  </widget>
  <widget class="QLabel" name="sentMessage">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 0); font-size:18px</string>
   </property>
   <property name="text">
    <string>Successfully Sent</string>
   </property>
  </widget>
  <widget class="QLabel" name="loginInvalidMessage">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0); font-size:18px</string>
   </property>
   <property name="text">
    <string>Invalid credential</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(167, 168, 167); 
color: rgb(255, 255, 255);
font-size:18px</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
